doc/sphinxman/source/build_planning.rst

```diff
@@ -895,6 +911,131 @@ old to compile |PSIfour|, but the Fortran compiler will work.
+    >>> cmake
+
+  .. code-block:: bash
+
+    >>> MATH_ROOT=/path/to/intel/vers/linux/mkl/ cmake
+
+E. Build with Intel MKL from conda
+
+  .. code-block:: bash
+
+    # won't work, as mkl.h header also needed
+
+F. OpenBLAS
+
+  .. code-block:: bash
+
+    >>> MATH_ROOT=/path/to/openblas/0.2.13_seq/x86_64/gcc_5.2.0/lib cmake
+
+G. Build with explicit MKL LAPACK
+
+  .. code-block:: bash
+
+    >>> cmake -DLAPACK_LIBRARIES="/path/to/lib/intel64/libmkl_lapack95_lp64.a;/path/to/lib/intel64/libmkl_rt.so" -DLAPACK_INCLUDE_DIRS="/path/to/mkl-h-include/"
+
+H. Build with explicit non-MKL LAPACK
+
+  .. code-block:: bash
+
+    >>> cmake -DLAPACK_LIBRARIES="/path/to/lib/liblapack.so;/path/to/lib/libblas.a"
+
+**Notes**
+
+* Much of |PSIfours| speed and efficiency depends on the corresponding
+  speed and efficiency of the linked BLAS and LAPACK libraries
+  (especially the former). Consider the following recommendations:
+
+  * It is NOT wise to use the stock BLAS library provided with many
+    Linux distributions like RedHat, as it is usually just the completely
+    unoptimized netlib distribution. The choice of LAPACK is less
+    critical, and so the unoptimized netlib distribution is acceptable.
+
+  * Perhaps the best choice, if available, is Intel's MKL library,
+    which includes efficient threaded BLAS and LAPACK (as of |PSIfour|
+    v1.1, earliest known working version is MKL 2013). On Mac, the
+    native Accelerate libraries are also recommended.
+
+  * For open-source LAPACK distributions, OpenBLAS (formerly GotoBLAS)
+    is known to work, while ATLAS is known
+    (https://github.com/psi4/psi4/issues/391) to have stability issues
+    with the DFOCC module.
+
+  * ACML libraries are known to work preferred with |PSIfour| v1.1 at
+    ACML 6.
+
+* The BLAS/LAPACK detected for |PSIfour| are also linked into any
+  Add-Ons (*e.g.*, libefp) that require them, rather than relying on
+  those packages' native math detection.
+
+* The separation between BLAS and LAPACK seen in detection printing
+  and CMake variables is purely formal. In practice, they get run
+  together and linked as ``${LAPACK_LIBRARIES} ${BLAS_LIBRARIES}``.
+
+* Sometimes the CMake's library search capabilites falter at SONAMEs
+  (*e.g.*, ``libblas.so.3`` *vs.* ``libblas.so``), extensions (static
+  *vs.* dynamic), or suffixes (*e.g.*, ``libacml_mp.so`` *vs.*
+  ``libacml.so``). The developers would be interested in hearing
+  of such problems to expand the math detection capabilities. The
+  immediate solution, however, is to form symlinks between the
+  library names that exist and the names expected. Consult file
+  :source:`cmake/math/MathLibs.cmake` for the library patterns being
+  sought.
+
+* The BLAS/LAPACK interface is standardized, so only libraries, not
+  headers, need to be detected. The exception is MKL, where the ``mkl.h``
+  header defines additional functionality; it must be located to use
+  BLAS threading.
+
```

Comment:
Thanks. I've been avoiding writing math detection docs for two years now, so my past few days' learning is now dumped to disk.